src/routes/Navbar.svelte

```diff
@@ -106,9 +99,9 @@
           {shopItemCounts}
         />
       {:else}
-        <button class="btn btn-ghost" on:click={() => signIn("keycloak")}>
+        <LoadingButton class="btn btn-ghost" onClick={() => signIn("keycloak")}>
           {m.navbar_logIn()}
-        </button>
+        </LoadingButton>
```

Comment:
Why is this a loading button? Does it make a difference when all `signIn` does is redirect the user?

Reply:
Not really, it actually does a fetch first. At least I noticed this quite clearly when I was on a slow network. I pressed the sign in button, it took about 2-3 seconds of nothing, then it navigated to the login page.

Reply:
Huh, I didn't know that. [You're right](https://github.com/nextauthjs/next-auth/blob/main/packages/frameworks-sveltekit/src/lib/client.ts#L42)!